utils/config/daos_server.yml

```diff
@@ -111,6 +111,14 @@
 #disable_vfio: true
 #
 #
+## Disable VMD Usage
+#
+## In some circumstances it may be preferable to not use Intel Volume
```

Comment:
Technically, this needs to be "Intel®". I think you could just omit it here for consistency.

Reply:
will change